apps/web/src/components/Dashboard/Team.tsx

```diff
@@ -62,7 +66,7 @@ export const TeamContent = ({
         `${process.env.NEXT_PUBLIC_BILLING_API_URL}/teams/${team.id}/users`,
         {
           headers: {
-            'X-Team-API-Key': team.apiKeys[0],
+            'X-User-Access-Token': user.accessToken,
```

Comment:
i assume the server side is aware of this header change?

Reply:
I would send both for now, add an issue to remove once the new backend is deployed

Reply:
after https://github.com/e2b-dev/billing/pull/25 is merged, yep!

Reply:
sending both headers now